docs/faq.md

```diff
@@ -45,25 +55,29 @@ jupytext --set-formats ipynb,md --execute *.md  # convert all .md files to paire
 
 ## Which files should I version control?
 
-Unless you want to version control the output cells, you should version the text file only (and add `*.ipynb` to `.gitignore`). As discussed above, Jupyter will let you open the text representation as a notebook and will re-create the `.ipynb` file when you save the notebook.
+Unless you want to version the outputs, you should version *only the text representation*. The paired `.ipynb` file can safely be deleted. It will be recreated locally the next time you open the notebook (from the text file) and save it.
```

Comment:
A lot of people also use notebooks as document previews in eg Github, where code cells have been executed and the readable document is the complete one. This speaks more to then having a separation of concerns, which different directory paths for paired documents can help with, between a document for version control, and a document for reading / display.